pkg/cluster/internal/providers/docker/provision.go

```diff
@@ -203,6 +204,11 @@ func commonArgs(cluster string, cfg *config.Cluster, networkName string, nodeNam
 		args = append(args, "-e", fmt.Sprintf("%s=%s", key, val))
 	}
 
+	if v, ok := os.LookupEnv("KIND_CONTAINERD_SNAPSHOTTER"); ok {
```

Comment:
I'm not sure this makes sense versus just having it in a version of the image. but if it does then the pattern is `KIND_EXPERIMENTAL_*` for enabling things that aren't stable yet.